tests/init.pp

```diff
@@ -9,4 +9,4 @@
 # Learn more about module testing here:
 # http://docs.puppetlabs.com/guides/tests_smoke.html
 #
-include mumble
```

Comment:
File could be moved to examples/init.pp (or removed altogether since we have rspec tests)?

If https://github.com/voxpupuli/modulesync_config/pull/210 is merged, this would be required for tests to pass.